server/graphql/v2/object/Order.js

```diff
@@ -235,6 +244,20 @@ export const Order = new GraphQLObjectType({
           }
         },
       },
+      hostCurrencyFxRate: {
+        type: GraphQLFloat,
+        description: 'The FX rate used to convert the amount to the host currency',
+        async resolve(order, _, { loaders }) {
+          const collective = order.collective || (await loaders.Collective.byId.load(order.CollectiveId));
+          const hostCollectiveId = collective?.HostCollectiveId;
+          if (hostCollectiveId) {
+            const host = await loaders.Collective.byId.load(hostCollectiveId);
+            return await getFxRate(order.currency, host.currency);
```

Comment:
Seems we now have a loader for fx rates, search for `loaders.CurrencyExchangeRate`.

Reply:
Thanks. I've changed this to use `loaders.CurrencyExchangeRate`.